docs/DeveloperGuide.md

```diff
@@ -154,10 +154,87 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
 This section describes some noteworthy details on how certain features are implemented.
 
-### \[Proposed\] Undo/redo feature
+### Add Gender
+
+The Add Gender feature allows users to add a gender field (Male / Female) to a person in the contact list. It is performed as a part 
+of `addCommand#execute()`.
+
+These operations are exposed in the `Model` interface as the method `Model#addPerson()`, which calls 
+`AddressBook#addPerson()` which calls `UniquePersonList#add` to add a new person in the person list 
+stored in AddressBook. 
+
+The following sequence diagram shows the methods calls related to add person operation: 
+
+![AddPersonSequenceDiagram](images/AddPersonSequenceDiagram.png)
```

Comment:
Thanks for all the suggestions! I will change the diagrams and add in the source files

Reply:
`AddPersonCommandParser()` method creates a new `AddPersonCommand` object. So the sequence diagram should show an arrow from the `AddPersonCommandParser` activation bar to the `AddPersonCommand` box instead of its activation bar.

Also there is an arrow from the `AddPersonCommand` activation bar pointing to the box. I think it should be a dotted line since that's the lifeline

![Capture](https://user-images.githubusercontent.com/92283593/197161450-37bd9802-9e02-4126-a493-829a8e3601e0.PNG)